metacat-main/src/main/java/com/netflix/metacat/main/services/notifications/sns/SNSNotificationPartitionAddMsg.java

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package com.netflix.metacat.main.services.notifications.sns;
+
+/**
+ * Enum class for partition add payload message.
+ *
+ * @author zhenl
+ * @since 1.2.0
+ */
+public enum SNSNotificationPartitionAddMsg {
+    /**
+     * Attached Valid Paritition Key.
+     */
+    AttachedValidParititionKey,
```

Comment:
Make the enum values upper case with underscores. 1. ATTACHED 2. INVALID_FORMAT, 3. ALL_FUTURE 3. MISSING_METADATA 5. DISABLED